contracts/governance/Vesting/VestingRegistryStorage.sol

```diff
@@ -61,4 +61,17 @@ contract VestingRegistryStorage is Initializable, AdminRole {
     ///@notice A record of all vesting addresses
     ///@dev isVesting[address] returns if the address is a vesting address
     mapping(address => bool) public isVesting;
+
+    /// @notice Store vesting creation type & vesting type information
+    /// @dev it is packed into 1 single storage slot for cheaper gas usage
+    struct VestingCreationAndTypeDetails {
+        bool isSet;
+        uint120 vestingType;
```

Comment:
this is weird. i prefer powers of 2 and i think that it's extremely unlikely that we'll ever have more than 2^32 types, so setting it to uint32 should be more than enough - for both types.

Reply:
Updated